app/helpers/view_components/site_footer.rb

```diff
@@ -18,5 +15,26 @@ def to_s
         end
       end
     end
+
+    private
+    def cache_key
```

Comment:
I took the liberty to slightly rewrite this as adding another clause was unwieldy